source/state_representation/CMakeLists.txt

```diff
@@ -26,82 +26,83 @@ set(CORE_SOURCES
   src/Geometry/Ellipsoid.cpp
```

Comment:
There is a command that gathers all cpp files from a directory. Could that be useful here? Or does this list exclude certain cpp files @buschbapti ?
```
file(GLOB helloworld_SRC
     "*.h"
     "*.cpp"
)

add_executable(helloworld ${helloworld_SRC})
```

Reply:
I would be happy to use recursive glob for gathering test sources together, since they have a common prefix. The potential catch with doing the same for the library sources is that it might also catch the test*.cpp files without additional filtering. In general smart use of file searching is not a bad thing.